design/one-pager-stacks-security-isolation.md

```diff
@@ -109,57 +141,57 @@ The diagram below summarizes the artifacts in the control plane after the GCP st
 
 ![Diagram of a Cluster Stack Installed](./images/cluster-stack-installed.png)
```

Comment:
the 2 images of cluster stacks and namespaced stacks are out of date now. do you have new architecture diagrams that demonstrate the proposed way of organizing roles/controllers/CRDs/etc.?

Reply:
updated